<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-LT5NG4U" ignored="3" name="Test(failed)" tests="1" failures="1" timestamp="2022-08-19T17:05:23 IST" time="0.069" errors="0">
  <testcase name="method3" time="0.0" classname="testNG.FirstTestngClass">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at testNG.FirstTestngClass.method3(FirstTestngClass.java:22)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- method3 -->
  <testcase name="method1" time="0.0" classname="testNG.FirstTestngClass">
    <ignored/>
  </testcase> <!-- method1 -->
  <testcase name="method4" time="0.0" classname="testNG.FirstTestngClass">
    <ignored/>
  </testcase> <!-- method4 -->
  <testcase name="method2" time="0.0" classname="testNG.FirstTestngClass">
    <ignored/>
  </testcase> <!-- method2 -->
</testsuite> <!-- Test(failed) -->
